terasoluna-gfw-functionaltest-selenium/src/test/resources/META-INF/spring/seleniumContext.xml

```diff
@@ -55,7 +55,7 @@
     </bean>
 
     <beans profile="firefox default">
-        <bean id="webDriver" class="org.openqa.selenium.firefox.FirefoxDriver"
+        <bean id="webDriver" class="org.terasoluna.gfw.functionaltest.app.FirefoxDriverFactoryBean"

```

Comment:
@iwanagah , Good catchup !! 
However, i think scope of factory bean should change to the `singleton` (omit the `scope="prototype"`).